addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketMessageDTO.java

```diff
@@ -249,4 +249,100 @@ public boolean isForceIntercept() {
     public Map<String, String> toEventData() {
         return this.toMap(true);
     }
+
+    public WebSocketChannelDTO getChannel() {
+        return channel;
+    }
+
+    public void setChannel(WebSocketChannelDTO channel) {
+        this.channel = channel;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Long getTimestamp() {
+        return timestamp;
+    }
+
+    public void setTimestamp(Long timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public String getDateTime() {
+        return dateTime;
+    }
+
+    public void setDateTime(String dateTime) {
+        this.dateTime = dateTime;
+    }
+
+    public Integer getOpcode() {
+        return opcode;
+    }
+
+    public void setOpcode(Integer opcode) {
+        this.opcode = opcode;
+    }
+
+    public String getReadableOpcode() {
+        return readableOpcode;
+    }
+
+    public void setReadableOpcode(String readableOpcode) {
+        this.readableOpcode = readableOpcode;
+    }
+
+    public Object getPayload() {
+        return payload;
+    }
+
+    public void setPayload(Object payload) {
+        this.payload = payload;
+    }
+
+    public Integer getCloseCode() {
+        return closeCode;
+    }
+
+    public void setCloseCode(Integer closeCode) {
+        this.closeCode = closeCode;
+    }
+
+    public Boolean getIsOutgoing() {
```

Comment:
The "get" should be removed (and the field renamed to `outgoing`). Setter should be updated accordingly.